### PR TITLE
Add sea ice project description

### PR DESCRIPTION
**Pull request by bobturneruk**

### CODE REVIEW

File: _project_descriptions/polar.md [hunk 1]
```diff
@@ -0,0 +1,9 @@
+---
+key: polar
+---
+
+The Polar Thematic Exploitation Portal ([Polar TEP](https://portal.polartep.io/ssoportal/pages/login.jsf)) is a platform that allows research software to be uploaded and run, accessing a reservoir of satellite imaging data without having to download enormous image files. It enables software to be run interactively by end users using a web graphical user interface, and chained together into workflows.
+
+Software for tracking the sea ice edge and icebergs had previously been written using [MatLab](https://uk.mathworks.com/products/matlab.html). In order to be deployed on Polar TEP, this needed to be converted into python and made to run in a Docker container.
```
Comment (willfurnass) [inline _project_descriptions/polar.md]:
```suggestion
Software for tracking the sea ice edge and icebergs had previously been written using [MATLAB](https://uk.mathworks.com/products/matlab.html). In order to be deployed on Polar TEP, this needed to be converted into Python and made to run in a Docker container.
```

File: _data/projects.csv [hunk 2]
```diff
@@ -15,7 +15,7 @@ id,key,title,long_title,tech_methods,rses,start,end,department,level,show
 19,,Reprohacks,Reprohacks,,Anna Krystalli,01/01/2020,31/03/2020,N8 ,2,1
 21,,RateSetter+,RateSetter+,,John Charlton,18/11/2019,17/03/2021,Mechanical Engineering,2,1
 22,,Prognostic: Cardiac MRI with ML,Prognostic: Cardiac MRI with ML,"python, version control, static analysis, GitHub","David Jones, Robert Turner, Will Furnass",01/10/2020,30/09/2021,Computer Science,2,1
-24,,Polar Portal,Polar Portal,"python, docker, matlab",Robert Turner,01/11/2019,01/02/2020,Geography,2,1
+24,polar,Polar Portal,Polar Portal: Sea ice edge detection,"python, docker, matlab",Robert Turner,01/11/2019,01/02/2020,Geography,1,1
```
Comment (willfurnass) [inline _data/projects.csv:18]: Add GIS?

File: _project_descriptions/polar.md (hunk 1 above)
Comment (willfurnass) [reply _project_descriptions/polar.md]: Could add something about role testing played in porting?

**Review by willfurnass — APPROVED** [review]
Added minor comments

File: _data/projects.csv (hunk 2 above)
Comment (bobturneruk) [reply _data/projects.csv:18]: Yes!

File: _project_descriptions/polar.md (hunk 1 above)
Comment (bobturneruk) [reply _project_descriptions/polar.md]: I hadn't adopted the test-driven approach I later used in GPy at this point.

**Comment by bobturneruk** [comment]
Thanks @willfurnass!

**Pull request merged**